src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -885,10 +889,6 @@ class ResourceService(
           for {
             originalPolicies <- accessPolicyDAO.listAccessPolicies(policyId.resource, samRequestContext)
             policyChanged <- accessPolicyDAO.setPolicyIsPublic(policyId, public, samRequestContext)
-            _ <- directoryDAO.updateGroupUpdatedDateAndVersionWithSession(
-              FullyQualifiedPolicyId(policyId.resource, policyId.accessPolicyName),
-              samRequestContext
-            )
```

Comment:
Note that the next thing that happens is onPolicyUpdateIfChanged which calls onPolicyUpdate if the policy actually changed.